### ISSUE TITLE
SpotifyiOS/SpotifyiOS.h

### ISSUE THREAD

**Issue by adrianosbr** [cjam/react-native-spotify-remote#25]
Thank you so much for this library and for the wonderful update :)

I created a new project just to test the library's new version and detected a problem. The headers for the Spotify SDK are now into a weird "Versions/A/Headers" folder and that prevents Xcode to build the app.

I just copied them directly to a "Headers" folder and now it works flawlessly ;)

**Comment by lucasgismondi** [comment]
+1 the great update! Also experiencing this issue.

**Comment by cjam** [comment]
Ah yes.  Are you using CocoaPods or just linking manually.  The CocoaPods handles all of this for you.  But I suppose a few docs about this in the manual linking section of the docs might be helpful.

**Comment by adrianosbr** [comment]
I used Cocoapods.

**Comment by cjam** [comment]
Hmm interesting, cuz this:
https://github.com/cjam/react-native-spotify-remote/blob/738566e1441d2e6482d221f646a96a756a7bf99a/RNSpotifyRemote.podspec#L16

Should take care of that part.  Did you do a `pod install` after updating?

**Comment by lucasgismondi** [comment]
I use cocoapods also and yes I did run pod install after the upgrade.

**Comment by cjam** [comment]
Hmm, I'll investigate when I integrate it into my own project.

**Comment by cjam** [comment]
Thanks for the report on this one.  

For those curious.  NPM publish follows but strips out symlinks from packages. 
 Apparently I had manually moved some files around in the SpotifySDK submodule (i.e. Headers, SpotifyiOS framework) that were published in 0.8.0 so everything worked fine, but when I updated to SDK 1.2 it reverted back to the symlinked structure.  Took me a while to figure out if it was an issue that could be solved with the podspec or something else.  

I've added some `prepack` `postpack` scripts to the `package.json` that make sure the Headers and Framework files are at the root right before publishing.  

Things should work without any modification to your project / pods project in `v0.1.1`
